main.tf

```diff
@@ -1,123 +1,167 @@
 module "origin_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.7"
-  namespace  = "${var.namespace}"
-  stage      = "${var.stage}"
-  name       = "${var.name}"
-  delimiter  = "${var.delimiter}"
-  attributes = ["${compact(concat(var.attributes, list("origin")))}"]
-  tags       = "${var.tags}"
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
+  namespace  = var.namespace
```

Comment:
please add `var.environment`, we are adding it to all our modules now.
See https://github.com/cloudposse/terraform-aws-eks-fargate-profile/blob/master/examples/complete/variables.tf#L21
https://github.com/cloudposse/terraform-aws-eks-fargate-profile/blob/master/variables.tf#L13
https://github.com/cloudposse/terraform-aws-eks-fargate-profile/blob/master/main.tf#L13